modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -33,7 +33,88 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
+
+Regular validations enable applications to compare a request with API specifications. If the proxy receives a query parameter or a header that is not specified in the definition, that parameter is irrespectively sent to the backend service. You can perform regular validations with WSDL and REST/RAML proxies. As part of the regular validations for REST/RAML proxies, the payload (schema), query parameters, URI parameters, headers, and form parameters are validated. For WSDL proxies, the envelope is validated against the structure provided in the WSDL file.
+
+
+=== Strict Validations
+
+Strict validations enable applications to compare a request with proxy specifications and allow only those headers and query parameters that are specified in the API definition. You can perform strict validations for only query parameters, only headers, or for both. This feature is currently available only for RAML and WSDL proxies in Mule 4. 
+
+The following tables describe the default settings for API proxies based on different Mule versions:
+
+
+==== Default Setting by API Proxy Versions for Mule 3.x Applications
+
+[%header%autowidth.spread]
+|===
+| Mule Version | API Proxy | Default Setting
+| 3.7x and below | RAML | Disabled 
+| 3.7x and below | WSDL | Disabled 
+| 3.8x and above | RAML | Disabled 
+| 3.8x and above | WSDL | Disabled 
+|===
+
+Mule 3.x applications and API proxy compatibility synopsis:
+
+* Only regular proxy validations are available for Mule 3.x applications.
+* Validations are performed for only WSDL and RAML proxies.
+* Proxies are not versioned in Mule 3.x applications.
+* You can edit proxies from API Manager for Mule 3.8.x and above.
+
+
+==== Default Setting by API Proxy Versions for Mule 4.x Applications
+
+[%header%autowidth.spread]
+|===
+| Mule Version | API Proxy | Default Setting
+| 4.x| RAML 1.1.3 and below  | Enabled 
+| 4.x| RAML 1.2.0            | Enabled 
+| 4.x| REST 2.0.0            | Enabled 
+| 4.x| REST 2.1.0 and above  | Enabled 
+| 4.x| WSDL 1.2.0 and below  | Disabled 
+| 4.x| WSDL 1.2.1 and above  | Disabled 
+|===
+
+Mule 4.x applications and API proxy compatibility synopsis:
+
+* For Mule 4.x applications, both regular and strict validations are available.
+* Validations are performed for only WSDL and RAML/REST proxies.
+* Proxies are versioned in Mule 4.x applications. 
+* You can edit RAML/REST proxies from API Manager for RAML proxy version 1.2.0 and REST proxy version 2.1.0 and above.
+
+
+=== API Specifications for Strict Validations
+
+When enabling strict validations for Mule 4 applications, you must consider your API architecture. If your Mule application has a proxy or a load balancer in front of it, the web server might modify the client request to include additional headers. In such cases, the Mule application denies the incoming request due to a non-valid header error. 
+
+
+==== Runtime Fabric headers
+
+The following headers are added by Edge in Runtime Fabric:
+
+`headers:
```

Comment:
Can we format this properly? Not sure if the docs page support it.

```
 x-correlation-id:
     type: string
   x-forwarded-for:
     type: string
   via:
     type: string
   x-forwarded-host:
     type: string
```

Reply:
That is how it is in the file, but when the content is rendered, it creates an issue.